runtime_config_linux.go

```diff
@@ -195,3 +198,12 @@ type Syscall struct {
 	Action Action `json:"action"`
 	Args   []*Arg `json:"args"`
 }
+
+type LinuxState struct {
```

Comment:
You're defining `State` in `config.go` (part of the `Spec` tree) and `LinuxState` in `runtime_config_linux.go` (part of the `RuntimeSpec` tree).  I'd [still prefer a separate `state.go` and `state_linux.go`][1], but if you want to [avoid adding more files][2], I'd shift `State` to `runtime_config.go`.

[1]: https://github.com/opencontainers/specs/pull/87#issuecomment-128137008
[2]: https://github.com/opencontainers/specs/pull/87#issuecomment-128137602